charts/tidb-operator/values.yaml

```diff
@@ -4,6 +4,9 @@
 # Also see rbac.create and controllerManager.serviceAccount
 clusterScoped: true
 
+# kubeVersion is the k8s cluster version
+kubeVersion: v1.10.2
```

Comment:
It seems almost guaranteed that this will not be set properly: is there any way to automatically get this setting?

Reply:
Yes, this version should be retrieved from current cluster. The variable `Capabilities.KubeVersion` is just for this purpose.
https://github.com/helm/helm/blob/master/docs/chart_template_guide/builtin_objects.md
> Capabilities.KubeVersion provides a way to look up the Kubernetes version. It has the following values: Major, Minor, GitVersion, GitCommit, GitTreeState, BuildDate, GoVersion, Compiler, and Platform.